documentation/book/getting-started.adoc

```diff
@@ -1,5 +1,12 @@
 == Getting started
 
+=== Downloading {ProductName}
+
+Strimzi releases are available for download from {ReleaseDownload}.
+The release artifacts contain documentation and example YAML files for deployment on {ProductPlatformName}.
+The example files are used through this documentation and can be used to install {ProductName}.
```

Comment:
Technically correct, but in my opinion, if you want to be even more explicit about specifying that example files are used in every part of the documentation then change through --> throughout. Hows that for Bostonian dialect ;)

Reply:
:-) Thanks. Should be fixed now.